patches/net-cookies-cookie_monster.cc.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/net/cookies/cookie_monster.cc b/net/cookies/cookie_monster.cc
+index 8e1134654e2b1db09f08f58036cc76c7978a7072..a0297fd573e61aa9fcf0c1f72c5df42c019e80f2 100644
+--- a/net/cookies/cookie_monster.cc
++++ b/net/cookies/cookie_monster.cc
+@@ -737,9 +737,13 @@ void CookieMonster::GetCookieListWithOptions(
+ 
+     if (!cookie_partition_key_collection.IsEmpty()) {
+       if (cookie_partition_key_collection.ContainsAllKeys()) {
++        std::vector<CookiePartitionKey> cookie_partition_keys;
```

Comment:
Are you not using BRAVE_ patches for readability?

Reply:
`BRAVE_` patches are a long term solution with our custom logic change. Here we don't change any logic, and I expect this to be cleaned up pretty soon with a proper upstream fix.

also a simple `BRAVE_` patch line here won't be enough, I still need to touch other lines.